src/StandardGovernor.sol

```diff
@@ -228,6 +228,11 @@ contract StandardGovernor is IStandardGovernor, BatchGovernor {
 
     /* ============ View/Pure Functions ============ */
 
+    /// @inheritdoc IGovernor
+    function COUNTING_MODE() external pure returns (string memory) {
+        return "support=against,for&quorum=for&success=standard";
```

Comment:
The `success` param is not something `COUNTING_MODE` defines. I made it up as a reasonable param. "standard" as a value ilpies how every other "normal" works: `yes >= no`. If there is a better name, maybe one that distinguishes between `yes > no` and `yes >= no`, it would be nice. `success=quorum` is something I also made up, but is reasonable to be interpreted as "if quorum is met, that is success."